# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid   ""
msgstr  "Project-Id-Version: vigilo.connector\n"
        "Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
        "POT-Creation-Date: 2010-02-19 10:16+0100\n"
        "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
        "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
        "Language-Team: LANGUAGE <LL@li.org>\n"
        "MIME-Version: 1.0\n"
        "Content-Type: text/plain; charset=CHARSET\n"
        "Content-Transfer-Encoding: 8bit\n"

#: src/vigilo/connector/nodetoqueuefw.py:91
msgid   "Incoming message is too big to be stored in the queue, dropping it."
msgstr  ""

#: src/vigilo/connector/nodetoqueuefw.py:96
#, python-format
msgid   "The queue is full, dropping incoming XML message! (%s)"
msgstr  ""

#: src/vigilo/connector/store.py:111
msgid   "The database is locked"
msgstr  ""

#: src/vigilo/connector/store.py:116
#, python-format
msgid   "Some operational error occured. Original error message was: %s"
msgstr  ""

#: src/vigilo/connector/store.py:125
#, python-format
msgid   "Some error occured. Original error message was: %s"
msgstr  ""

#: src/vigilo/connector/converttoxml.py:37
msgid   "Got one-to-one message"
msgstr  ""

#: src/vigilo/connector/converttoxml.py:42
msgid   "Got empty line"
msgstr  ""

#: src/vigilo/connector/converttoxml.py:44
msgid   "Got 'event' message"
msgstr  ""

#: src/vigilo/connector/converttoxml.py:47
msgid   "Got 'perf' message"
msgstr  ""

#: src/vigilo/connector/converttoxml.py:50
msgid   "Got 'downtime' message"
msgstr  ""

#: src/vigilo/connector/converttoxml.py:53
msgid   "Got 'command' message"
msgstr  ""

#: src/vigilo/connector/converttoxml.py:58
#: src/vigilo/connector/converttoxml.py:65
#: src/vigilo/connector/converttoxml.py:71
#, python-format
msgid   "Unknown/malformed message type: '%s'"
msgstr  ""

#: src/vigilo/connector/converttoxml.py:75
#, python-format
msgid   "Got malformed message: %s"
msgstr  ""

#: src/vigilo/connector/queuetonodefw.py:81
#, python-format
msgid   "Error callback in consumeQueue(): %s"
msgstr  ""

#: src/vigilo/connector/queuetonodefw.py:101
msgid   "Received request to shutdown"
msgstr  ""

#: src/vigilo/connector/queuetonodefw.py:109
msgid   "Item is None in consumeQueue, this should never happen!"
msgstr  ""

#: src/vigilo/connector/queuetonodefw.py:118
msgid   "Stopping QueueToNodeForwarder."
msgstr  ""

#: src/vigilo/connector/sockettonodefw.py:109
#, python-format
msgid   "Received message: %r"
msgstr  ""

#: src/vigilo/connector/sockettonodefw.py:152
#: src/vigilo/connector/sockettonodefw.py:191
#, python-format
msgid   "Message from Socket impossible to forward (no connection to XMPP "
        "server), the message is stored for later reemission (%s)"
msgstr  ""

#: src/vigilo/connector/sockettonodefw.py:172
#, python-format
msgid   "Unable to forward the message (%(error)r), it has been stored for "
        "later retransmission (%(xml_src)s)"
msgstr  ""

#: src/vigilo/connector/main.py:65
#, python-format
msgid   "Directory not found: '%(dir)s'"
msgstr  ""

#: src/vigilo/connector/main.py:70
#, python-format
msgid   "Directory not readable: '%(dir)s'"
msgstr  ""

#: src/vigilo/connector/main.py:75
#, python-format
msgid   "Directory not writable: '%(dir)s'"
msgstr  ""

#: src/vigilo/connector/main.py:80
#, python-format
msgid   "Directory not executable: '%(dir)s'"
msgstr  ""

#: src/vigilo/connector/nodetosocketfw.py:43
msgid   "ConnectionInitialized"
msgstr  ""

#: src/vigilo/connector/nodetosocketfw.py:120
msgid   "Message impossible to forward (socket not connected), the message is "
        "stored for later reemission"
msgstr  ""

#: src/vigilo/connector/nodetosocketfw.py:146
#, python-format
msgid   "Chat message to forward: '%s'"
msgstr  ""

#: src/vigilo/connector/nodetosocketfw.py:175
#, python-format
msgid   "Message from BUS to forward: %s"
msgstr  ""
